confignode/src/main/java/org/apache/iotdb/confignode/persistence/executor/ConfigRequestExecutor.java

```diff
@@ -44,10 +44,23 @@
 import org.apache.iotdb.confignode.persistence.ClusterSchemaInfo;
 import org.apache.iotdb.confignode.persistence.NodeInfo;
 import org.apache.iotdb.confignode.persistence.PartitionInfo;
+import org.apache.iotdb.confignode.persistence.Snapshot;
 import org.apache.iotdb.consensus.common.DataSet;
 
+import org.apache.thrift.TException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicBoolean;
+
 public class ConfigRequestExecutor {
```

Comment:
should implement Snapshot interface?

Reply:
`ConfigRequestExecutor` should only be an executor with no ability to process snapshots _(It is only responsible for scheduling)_ .
The module that implements the Snapshot interface should be able to operate the snapshot itself. Such as `ClusterSchemaInfo`

Reply:
OK

Reply:
OK